concepts/05 UI Components/Lookup/00 Getting Started with Lookup/25 Group Data.md

```diff
@@ -0,0 +1,101 @@
+You can group data by category in the Lookup. To implement this, we will use the data from the previous steps with the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) component that allows you to sort, filter, select, and group data.
+
+To group data, specify the data field to group by the **DataSource**'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property and set the Lookup's [grouped](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#grouped) property to **true**.
+
+If the number of items doesn't fit in the Lookup size, the scrolling bar appears. You can load items by parts on the "next" button click. Follow the steps below to enable this functionality:
```

Comment:
Paging is a different feature, it's not part of grouping.

Reply:
Noted